packages/salesforcedx-vscode-apex-replay-debugger/coverconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
When these changes stablilize, could you please add an explanation of these configurations to our testing documenation?

Reply:
Yup, definitely will update the documentation!